sessions/testing.qmd

```diff
@@ -161,6 +162,19 @@ TODO: Reading text
 #| code-summary: "**Click for the solution**. Only click if you are struggling or are out of time."
 ```
 
+TODO: Reading text
+
+## Exercise: TODO: Run a CRAN check on the package
```

Comment:
As said above, we would ideally run this with them the first time. The exercise would be to do it on their own, after doing some changes or improving things.